### PR TITLE
bugfix: fix identation for options.yml code snippet

### PR DESCRIPTION
**Pull request by rahulreddy**

### CODE REVIEW

File: docs/Installation/Installation_Guide.rst [hunk 1]
```diff
@@ -492,29 +492,28 @@ parameters as appropriate.)
...
-     enabled: "true"
-     annotations:
-       nginx.ingress.kubernetes.io/proxy-body-size: 0
-
-    hosts:
-      -  zenko.local
+      enabled: "true"
+      annotations:
+        nginx.ingress.kubernetes.io/proxy-body-size: 0
+      hosts:
```
Comment (JianqinWang) [inline docs/Installation/Installation_Guide.rst:498]: Is `hosts` supposed to be a sub-item of `ingress`? or is it on the same level as `ingress`

Comment (rahulreddy) [reply docs/Installation/Installation_Guide.rst:498]: It's supposed to go under ingress. That's the actual fix.

**Review by tmacro — APPROVED** [review]

**Review by dora-korpar — APPROVED** [review]

**Review by JianqinWang — APPROVED** [review]

**Review by anurag4DSB — APPROVED** [review]

File: docs/Installation/Installation_Guide.rst [hunk 2]
```diff
@@ -492,29 +492,28 @@ parameters as appropriate.)
    ::
 
     ingress:
-     enabled: "true"
-     annotations:
-       nginx.ingress.kubernetes.io/proxy-body-size: 0
-
-    hosts:
-      -  zenko.local
+      enabled: "true"
```
Comment (bennettbuchanan) [inline docs/Installation/Installation_Guide.rst:495]: I see indentation levels in this file as both a single and double space. For example: [here](https://github.com/scality/Zenko/pull/268/files#diff-c22da790284d7726aa094bb10c9def89R215) it is still single space indentation.

File: docs/Installation/Installation_Guide.rst [hunk 3]
```diff
@@ -492,29 +492,28 @@ parameters as appropriate.)
...
 
 2. If your Zenko instance is behind a proxy, append the following
-   lines to the options.yml file, substituting your proxy’s IP addresses
-   and port assignments:
+   lines to the options.yml file, under the section **cloudserver**, substituting
+   your proxy’s IP addresses and port assignments:
 
    ::
 
       proxy:
```
Comment (giacope) [inline docs/Installation/Installation_Guide.rst:513]: Isn't there still an indentation problem here?

It should look like this:
```
proxy:
	http:
	...
```

File: docs/Installation/Installation_Guide.rst [hunk 4]
```diff
@@ -492,29 +492,28 @@ parameters as appropriate.)
...
       http: ""
       https: ""
-        caCert: false
+      caCert: false
```
Comment (giacope) [inline docs/Installation/Installation_Guide.rst]: Maybe it's a good moment to include the recently added `no_proxy` field to this doc.

**Review by giacope — CHANGES_REQUESTED** [review]

File: docs/Installation/Installation_Guide.rst (hunk 2 above)
Comment (rahulreddy) [reply docs/Installation/Installation_Guide.rst:495]: Yeah, I don't understand why. The metalk8s section will be removed anyways.

File: docs/Installation/Installation_Guide.rst (hunk 3 above)
Comment (rahulreddy) [reply docs/Installation/Installation_Guide.rst:513]: Right, I wasn't paying attention... I updated the commit with the correct indent.

File: docs/Installation/Installation_Guide.rst (hunk 4 above)
Comment (rahulreddy) [reply docs/Installation/Installation_Guide.rst]: Added.

**Review by giacope — APPROVED** [review]

**Comment by rahulreddy** [comment]
Closing this in favor of https://github.com/scality/Zenko/pull/270 as there is a typo in branch name `bugix`

**Pull request closed without merging**